lib/client/overlays/progress.js

```diff
@@ -45,7 +45,7 @@ const css = `
 }
 
 #${ns}-percent-value {
-  alignment-baseline: middle;
+  dominant-baseline: middle;
```

Comment:
also fixed alignment on the status overlay, it was giving me OCD :rofl:

Reply:
What's the explanation for this change?